catalog/resource_sql_table.go

```diff
@@ -332,6 +339,23 @@ func (ti *SqlTableInfo) diff(oldti *SqlTableInfo) ([]string, error) {
 		statements = append(statements, fmt.Sprintf("ALTER %s %s SET TBLPROPERTIES (%s)", typestring, ti.SQLFullName(), ti.serializeProperties()))
 	}
 
+	if !reflect.DeepEqual(ti.Tags, oldti.Tags) {
```

Comment:
What happens in the case of renaming of tags (not sure if it's supported at backend).